dev/integration_tests/windows_startup_test/lib/main.dart

```diff
@@ -38,14 +41,22 @@ void main() async {
     const MethodChannel methodChannel =
         MethodChannel('tests.flutter.dev/windows_startup_test');
 
-    final bool? visible = await methodChannel.invokeMethod('isWindowVisible');
+    final bool? honorsDarkMode = await methodChannel.invokeMethod<bool?>(
+      'getWindowAttributeBool',
+      <String, dynamic>{'attribute': kUseImmersiveDarkModeAttribute},
+    );
```

Comment:
Consider sticking this in a private method `Future<bool> _honorsDarkMode` method.

Reply:
Done!